### PR TITLE
Added hot reloading for functional components and refactored dumb components to functional style

### PR DESCRIPTION
**Pull request by sdeleon28**
Added hot reloading for functional components and refactored dumb components to functional style

### CODE REVIEW

File: app/containers/HomePage.js [hunk 1]
```diff
@@ -1,10 +1,6 @@
...
 import Home from '../components/Home';
 
-export default class HomePage extends Component {
-  render() {
-    return (
-      <Home />
-    );
-  }
-}
+export default () => (
```
Comment (amilajack) [inline app/containers/HomePage.js]: Wouldn't this be better?

```export default () => <Home />
```

Comment (amilajack) [inline app/containers/HomePage.js]: Wouldn't this be better?

```js
export default () => <Home />
```

Comment (sdeleon28) [inline app/containers/HomePage.js]: Sure, if we add a semicolon ;-) I'll do that in a sec

Comment (sdeleon28) [inline app/containers/HomePage.js]: Looks like one of the tests times out without the parentheses in Node 5. I'll try with the parentheses but inline.

**Comment by chentsulin** [comment]
Superseded by #421

**Pull request closed without merging**